SYCL/Basic/parallel_for_disable_range_roundup.cpp

```diff
@@ -0,0 +1,62 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -DSYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE=1 %t.out %GPU_CHECK_PLACEHOLDER --check-prefix CHECK-DISABLED
```

Comment:
Does it work for GPU only?
If so you need to set REQUIRES: gpu
Otherwise you need to add run lines for HOST/CPU/ACC

Is it supported on CUDA/Level_Zero backends. The test should be configured accordingly.